addons/missionmodules/ACE_Settings.hpp

```diff
@@ -0,0 +1,42 @@
+class ACE_Settings {
+    class GVAR(MaxBreathingDistance) {
+        typeName = "SCALAR";
+        value = 100;
+        isClientSettable = 1;
+        category = "Mix";
```

Comment:
Category would preferably be stringtabled in it's own module if it's used only in that module or in [`optionsmenu`](https://github.com/acemod/ACE3/blob/master/addons/optionsmenu/stringtable.xml) if used in more modules that aren't dependent on each other.

I'd say category name in this case should be either `Visual` or `Other`.